modules/govuk_jenkins/templates/jobs/deploy_app.yaml.erb

```diff
@@ -42,6 +42,27 @@
 
             git clone ${APP_DEPLOYMENT_GIT_URL} --branch master --single-branch --depth 1 ./
             ./jenkins.sh
+        - conditional-step:
+            condition-kind: and
+            condition-operands:
+              - condition-kind: or
+                condition-operands:
+                  <% @deploy_downstream_applications.keys.each do |app| %>
+                  - condition-kind: strings-match
+                    condition-string1: "$TARGET_APPLICATION"
+                    condition-string2: "<%= app %>"
+                  <% end %>
+                  - condition-kind: never
+              - condition-kind: regex-match
+                regex: "release_.*"
+                label: "$TAG"
+              - condition-kind: strings-match
+                condition-string1: "$DEPLOY_TASK"
+                condition-string2: "deploy"
+            steps:
+              - trigger-builds:
+                  - project: Deploy_App_Downstream
+                    current-parameters: true
```

Comment:
Perhaps worth clarifying this with a comment too:

```yaml
                    current-parameters: true # pass all of this job's build parameters to the downstream build
```

I just don't find the jenkins-job-builder config options particularly self-documenting!

Reply:
This parameter is pretty standard for the plugin, so I'm not going to comment here; it's covered by the documentation.